warcio/limitreader.py

```diff
@@ -67,3 +87,86 @@ def wrap_stream(stream, content_length):
 
         return stream
 
+    def configure_digesters(self, rec_type, segment_number, payload_digest, block_digest):
```

Comment:
If moved to separate class, perhaps this could just in the constructor and raise if nothing to verify?